firebase-common/src/androidMain/kotlin/dev/gitlive/firebase/_decoders.kt

```diff
@@ -47,3 +50,22 @@ private fun makeJavaReflectionDecoder(jvmObj: Any): CompositeDecoder {
         }
     }
 }
+
+private val geoPointKeys = setOf("latitude", "longitude")
```

Comment:
Declared multiple times, can be moved to common

Reply:
It belongs to platform implementation and in this particular case they're the same

Reply:
But is created for each platform implementation and is the same